packages/flutter/test/gestures/gesture_binding_resample_event_test.dart

```diff
@@ -0,0 +1,186 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:ui' as ui;
+
+import 'package:clock/clock.dart';
+import 'package:fake_async/fake_async.dart';
+import 'package:flutter/foundation.dart';
+import 'package:flutter/gestures.dart';
+import 'package:flutter/scheduler.dart';
+
+import '../flutter_test_alternative.dart';
+
+typedef HandleEventCallback = void Function(PointerEvent event);
+
+class TestResampleEventFlutterBinding extends BindingBase with GestureBinding, SchedulerBinding {
+  HandleEventCallback? callback;
+  FrameCallback? postFrameCallback;
+  Duration? frameTime;
+
+  @override
+  void handleEvent(PointerEvent event, HitTestEntry entry) {
+    super.handleEvent(event, entry);
+    if (callback != null)
+      callback?.call(event);
+  }
+
+  @override
+  Duration get currentSystemFrameTimeStamp {
+    assert(frameTime != null);
+    return frameTime!;
+  }
+
+  @override
+  int addPostFrameCallback(FrameCallback callback) {
+    postFrameCallback = callback;
+    return 0;
+  }
+
+  @override
+  SamplingClock? get debugSamplingClock => TestSamplingClock();
+}
+
+class TestSamplingClock implements SamplingClock {
+  @override
+  DateTime now() => clock.now();
+
+  @override
+  Stopwatch stopwatch() => clock.stopwatch();
+}
+
+typedef ResampleEventTest = void Function(FakeAsync async);
+
+void testResampleEvent(String description, ResampleEventTest callback) {
+  test(description, () {
+    fakeAsync((FakeAsync async) {
+      callback(async);
+    }, initialTime: DateTime.utc(2015, 1, 1));
+  }, skip: isBrowser);
```

Comment:
Can you add the reason for skipping web tests here?

Reply:
Done